app/src/main/res/drawable/profile_recyclerview_item_decor.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+  <size android:height="1dip" />
+  <solid android:color="#707070" />
+  <padding
+    android:bottom="48dp"
+    android:top="48dp" />
+</shape>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done.